python/paddle/tensor/logic.py

```diff
@@ -1415,6 +1415,48 @@ def bitwise_not_(x: Tensor, name: str | None = None) -> Tensor:
         return _C_ops.bitwise_not_(x)
 
 
+def bitwise_invert(
+    x: Tensor, out: Tensor | None = None, name: str | None = None
+) -> Tensor:
+    r"""
+    Apply ``bitwise_not`` (bitwise inversion) on Tensor ``x``.
+
+    This is an alias to the ``paddle.bitwise_not`` function.
+
+    .. math::
+        Out = \sim X
+
+    Note:
+        ``paddle.bitwise_invert`` is functionally equivalent to ``paddle.bitwise_not``.
+        It supports broadcasting, and you can refer to `Introduction to Tensor`_
+        for more information on broadcasting.
+
+        .. _Introduction to Tensor: ../../guides/beginner/tensor_en.html#chapter5-broadcasting-of-tensor
+
+    Args:
+        x (Tensor): Input Tensor of ``bitwise_invert``. It is a N-D Tensor of bool, uint8, int8, int16, int32, int64.
+        out (Tensor|None, optional): Result of ``bitwise_invert``. It is a N-D Tensor with the same data type as the input Tensor. Default: None.
+        name (str|None, optional): The default value is None. This property is typically not set by the user.
+            For more information, please refer to :ref:`api_guide_Name`.
+
+    Returns:
+        Tensor: Result of ``bitwise_invert``. It is a N-D Tensor with the same data type as the input Tensor.
+
+    Examples:
+        .. code-block:: python
+
+            >>> import paddle
+            >>> x = paddle.to_tensor([-5, -1, 1])
+            >>> res = paddle.bitwise_invert(x)
```

Comment:
这个样例应该是调用`x.bitwise_invert`
```suggestion
            >>> res = x.bitwise_invert()
```

Reply:
这两个实际运行起来中间会有什么细微差别吗。
我记得torch那边也是，一个tensor一个Tensor的，效果一样，写法不同。

Reply:
这两个实际运行起来中间会有什么细微差别吗。
我记得torch那边也是，一个tensor一个Tensor的，效果一样，写法不同。

Reply:
一个是Tensor类的方法，一个是paddle模块下的函数，Python的接口不同调用方式不同，底层都是调用同一个kernel